pkg/endpointmanager/manager.go

```diff
@@ -630,6 +630,9 @@ func (mgr *EndpointManager) AddHostEndpoint(ctx context.Context, owner regenerat
 		return err
 	}
 
+	node.SetEndpointID(ep.GetID())
+	log.Infof("Setting host endpoint ID to %d (%d)", node.GetEndpointID(), ep.GetID())
```

Comment:
Should it be logged also in ` pkg/endpoint/restore.go:184`?

Reply:
Whoops, I left this line by mistake. I used it to debug at some point. I don't think it's necessary as the host endpoint ID can already be retrieved from a couple places (`cilium endpoint list`, header files, etc.).

Fixed.